.github/actions/install-iceoryx-deps-and-clang/action.yml

```diff
@@ -5,14 +5,14 @@ runs:
     - name: Install iceoryx dependencies and clang-tidy
       shell: bash
       run: |
+        sudo wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+        sudo add-apt-repository "deb http://apt.llvm.org/focal/ llvm-toolchain-focal main"
         sudo apt-get update
         sudo apt-get install -y libacl1-dev libncurses5-dev
-        wget https://apt.llvm.org/llvm.sh
-        chmod +x llvm.sh
-        sudo ./llvm.sh 15 all ## uses clang version 15
+        sudo apt-get install -y clang-format-16 clang-tidy-16 clang-tools-16 clang-16
```

Comment:
Should we create an issue to downgrade to clang-tidy 15 once it is released?

Reply:
I would say that we will see what the 16 version brings, currently I see no need to downgrade it.